src/utils/SiriusAdapter.cpp

```diff
@@ -234,7 +234,11 @@ class TOPPSiriusAdapter :
     }
 
     // log in to Sirius account if email and password are specified
-    algorithm.logInSiriusAccount(sirius_executable, sirius_user_email, sirius_user_password);
+    if (!sirius_user_email.empty() && !sirius_user_password.empty())
+    {
+      algorithm.logInSiriusAccount(sirius_executable, sirius_user_email, sirius_user_password);
+    }
```

Comment:
Is this a warning or something we can‘t recover from? If we absolutely need a login then throwing an exception and terminating the program is bettet

Reply:
Unfortunately, it is very unclear when login is needed. Even the Sirius developers don't know it (at least we get Exceptions for searching in specific libraries, although this should be caught when starting Sirius already.)

Reply:
It is definitely not needed all the time.

Reply:
You need to be logged in, however you can also do that before and independent of SiriusAdapter. A successful login will be saved and is not necessary on subsequent runs. So it should continue if no login information was given.

Reply:
Ah ok thanks for the clarification.